aegis-web/yo/app/navProject.html

```diff
@@ -93,8 +93,15 @@
           Members
         </a>
       </li>
+      
+      <li ng-click="projectCtrl.goToProject()" ng-show="projectCtrl.cloak">
+        <a>
+          <span class="sub_icon fa fa-list fa-lg" style="width: 30px; display: inline-block; margin-right: 10px"></span>
+          Activity-Stream
+        </a>
+      </li>
 
-      <!-- li ng-click="projectCtrl.goToMetadataDesigner()" ng-show="projectCtrl.role === 'Data owner'" ng-show="projectCtrl.cloak">
+      <!-- li ng-click="projectCtrl.goToMetadataDesignuser()" ng-show="projectCtrl.role === 'Data owner'" ng-show="projectCtrl.cloak">
```

Comment:
this isn't correct.

Reply:
I didn't touch this line, I thought it came from another change. what is the correct one?

Reply:
I think it's a typo error. I corrected it

Reply:
`goToMetadataDesigner` like it's on the red line here on github